scaleway/resource_k8s_cluster_beta.go

```diff
@@ -385,6 +385,10 @@ func resourceScalewayK8SClusterBetaDefaultPoolRead(d *schema.ResourceData, m int
 	defaultPool["updated_at"] = pool.UpdatedAt.String()
 	defaultPool["status"] = pool.Status.String()
 
+	if pool.PlacementGroupID != nil {
+		defaultPool["placement_group_id"] = newRegionalId(region, *pool.PlacementGroupID)
```

Comment:
placement_group_id should not be regional

Reply:
forgot to push the commit :sweat_smile: